syntax="proto2";
package nicer;

option cc_generic_services = true;

message HttpRequest {};
message HttpResponse {};

service HttpService {
    rpc check_excel_status(HttpRequest) returns (HttpResponse);
};